Mathlib/Algebra/Polynomial/Roots.lean

```diff
@@ -199,16 +199,22 @@ lemma roots_neg (p : R[X]) : (-p).roots = p.roots := by
   rw [← neg_one_smul R p, roots_smul_nonzero p (neg_ne_zero.mpr one_ne_zero)]
 
 @[simp]
-theorem roots_C_mul_X_sub_C_of_IsUnit (b : R) (a : Rˣ) : (C (a : R) * X - C b).roots =
+theorem roots_C_mul_X_sub_C_of_isUnit (b : R) (a : Rˣ) : (C (a : R) * X - C b).roots =
```

Comment:
strictly this should be
```suggestion
theorem roots_C_unitsVal_mul_X_sub_C (b : R) (a : Rˣ) : (C (a : R) * X - C b).roots =
```
or at least `of_unit`; this doesn't mention `IsUnit` at all.